.travis.yml

```diff
@@ -2,13 +2,18 @@ language: php
 
 matrix:
     include:
-        - php: 5.6
-        - php: 7.0
-          env: lint=1
         - php: 7.1
           env: DEPENDENCIES='dev' lint=1
         - php: 7.1
           env: DEPENDENCIES='low'
+        - php: 7.2
+          env: DEPENDENCIES='dev' lint=1
+        - php: 7.2
+          env: DEPENDENCIES='low'
+        - php: 7.3
+          env: DEPENDENCIES='dev' lint=1
+        - php: 7.3
+          env: DEPENDENCIES='low'
```

Comment:
running `DEPENDENCIES='low'` and `DEPENDENCIES='dev'` looks useless to me. It increases the Travis test matrix (making it more likely to reach the maximum allowed concurrency).

And in this setup, there is no job running unmodified deps anymore btw (as 5.6 and 7.0 were the one doing it before)

Reply:
I think that `low` and `dev` help us to test it better.
Should I add build with unmodified deps for every php version?
@sstok What do you think?

Reply:
well, having 1 job for low deps is useful to ensure that lowest bounds are OK.
having 1 job for dev deps is useful to catch issues with upcoming version (getting a chance to fix them before they reach a release).
But the jobs using deps without modifying them are the most important ones: that's how the package will be used by most users (a dev deps is used only if you explicitly allow it in composer, and a lower dep is used only if some conflict forbids using the latest release)

Reply:
Updated

Reply:
you still have lots of tests with modified deps (meaning that a single build of this matrix already outweights the concurrency quota of the github organization, and has to wait on itself to run)

Reply:
So what do you propose? Leave dev and low only for one php version (for example for php 7.2 ) and remove dev and low for other versions?

Reply:
yes, exactly.

Reply:
and this is how it is currently in master btw: only one job for each of them.

Reply:
Updated

Reply:
@stof Is it OK now?

Reply:
@stof Do you have more suggestions?